README.md

```diff
@@ -259,7 +260,8 @@ GET `/v1/ecs/{account}/params/{prefix}/{param}`
             "Value": "someprefix"
         }
     ],
-    "LastModifiedDate": "2019-09-26 18:08:38 +0000 UTC"
+    "LastModifiedDate": "2019-10-09 15:43:44 +0000 UTC",
+    "Version": 1
```

Comment:
We should update this route in the beginning of the Readme:
`POST /v1/ecs/{account}/params`